ui/package.json

```diff
@@ -29,6 +29,7 @@
     "start": "react-scripts start",
     "serve": "react-scripts start",
     "build": "react-scripts build",
+    "postbuild": "tar -C build -czvf maas-ui-v$npm_package_version.tar.gz .",
```

Comment:
Should this get built into dist/ or something and then get added to the .gitignore?